controllers/organization_validator.go

```diff
@@ -74,19 +73,15 @@ func (v *OrganizationValidator) handle(ctx context.Context, req admission.Reques
 		}
 	}
 
-	orgClusters := &unstructured.UnstructuredList{}
-	orgClusters.SetGroupVersionKind(schema.GroupVersionKind{
-		Group:   "cluster.x-k8s.io",
-		Version: "v1alpha2",
-		Kind:    "cluster",
-	})
+	orgClusters := capi.ClusterList{}
 	{
-		err := v.List(ctx, orgClusters, &client.ListOptions{
+		err := v.List(ctx, &orgClusters, &client.ListOptions{
 			LabelSelector: orgClustersSelector,
 		})
 		if apimeta.IsNoMatchError(err) {
 			// If the CRD is not in place, just ignore the error.
-		} else if err != nil {
+		}
+		if err != nil {
```

Comment:
```suggestion
		} else if err != nil {
```

Otherwise it will still return an error

Reply:
this is what happens when I edit code while looking at people's desks. Thanks.